packages/itwinui-css/src/bridge.scss

```diff
@@ -245,35 +282,117 @@
+      fill: var(--ids-color-icon-neutral-disabled);
+    }
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Label */
+
+  // StrataKit labels are slightly muted
+  .iui-input-label:not([data-iui-disabled='true']) {
+    --_iui-input-label-color: var(--ids-color-text-neutral-secondary);
   }
 
   /* ---------------------------------------------------------------------------- */
-  /* ProgressRadial */
+  /* ProgressLinear, ProgressRadial */
 
   // StrataKit progress indicators's default color is not accent
   :is(.iui-progress-indicator-linear, .iui-progress-indicator-radial) {
     --_iui-progress-indicator-color-default-progress: var(--ids-color-icon-neutral-base);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Select */
+
+  // StrataKit Select background is not --iui-color-background's global mapping.
+  .iui-select-button {
+    --_iui-field-color-background: var(--ids-color-bg-neutral-base);
+    --_iui-field-color-background-hover: var(--_iui-field-color-background);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* SideNavigation */
+
+  // iTwinUI background color (--iui-color-background-backdrop)'s global mapping doesn't look correct in this context
+  .iui-side-navigation
+    .iui-sidenav-button:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true']) {
+    background-color: var(--iui-color-background-disabled);
+  }
+
   /* ---------------------------------------------------------------------------- */
   /* Slider */
-  /* TODO: Since Slider is experimental in StrataKit, revisit this when stable on the StrataKit end */
-  /* TODO: Are minLabel and maxLabel fills/colors correct? */
+  /* TODO: Since StrataKit Slider tokens are not added, revisit this when those tokens are added on the StrataKit end */
+
+  // Track color is not --iui-color-border-accent's global mapping
+  .iui-slider {
+    --_iui-slider-track-color: var(--ids-color-bg-accent-base);
+  }
 
   // Background and border is not iui-color-background and iui-color-border-foreground respectively
   .iui-slider-thumb {
-    background-color: var(--ids-color-bg-neutral-faded);
+    background-color: var(--ids-color-bg-neutral-inverse);
     border-color: var(--ids-color-border-neutral-faded);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Surface */
+
+  .iui-surface {
+    // To match the surface similar to StrataKit's dropdown menu
+    background-color: var(--ids-color-bg-elevation-base);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Table */
+
+  // StrataKit table and unselected rows match the backdrop color
+  :is(.iui-table, .iui-table-row:not([aria-selected='true']) .iui-table-cell, .iui-table-paginator, .iui-table-empty) {
+    background-color: var(--iui-color-background-backdrop);
```

Comment:
Where are you seeing that it table rows match the backdrop? That doesn't sound right.

Reply:
Both the comment and the mapping is wrong. It should just be `--iui-color-background` which (1) is already the background used by table, and (2) also maps to `bg-page-base`. So, this entire thing can be removed.